reviews/2022-Network Service Mesh-annual.md

```diff
@@ -0,0 +1,171 @@
+* [v1.0.0](https://networkservicemesh.io/docs/releases/v1.0.0/) - Jun 19, 2021
+* [v1.1.0](https://networkservicemesh.io/docs/releases/v1.1.0/) - Nov 28, 2021
+* [v1.2.0](https://networkservicemesh.io/docs/releases/v1.2.0/) - Feb 15, 2021
+
+## Annual Review Questions
+
+1.  _Include a link to your project’s devstats page. We will be looking for signs of consistent or increasing contribution activity. Please feel free to add commentary to add colour to the numbers and graphs we will see on devstats._
+
+    [Network Service Mesh Devstats](https://networkservicemesh.devstats.cncf.io/)
+    
+    Highlights:
+    *  [30 contributors](https://networkservicemesh.devstats.cncf.io/d/22/prs-authors-table?orgId=1&var-period_name=Last%20year&var-repogroup_name=All)
+    *  [9 Organizations](https://networkservicemesh.devstats.cncf.io/d/5/companies-table?orgId=1&var-period_name=Last%20year&var-metric=commits)
+       
+       Including:
+       *  Cisco
+       *  Ericsson Software Technology
+       *  Red Hat
+    * [Commits](https://networkservicemesh.devstats.cncf.io/d/2/commits-repository-groups?orgId=1&from=now-1y&to=now&var-period=d7&var-repogroups=All)
+      have remainded strong in the last year.
+    
+2.  _How many maintainers do you have, and which organisations are they from? (Feel free to link to an existing MAINTAINERS file if appropriate.)_
+
+    Network Service Mesh has five top level MAINTAINERS:
+    * Andre Sobolov (Xored)
+    * Ed Warnicke (Cisco)
+    * Frederick Kautz (Anthem)
+    * Nikolay Nikolaev (Juniper)
+    * Denis Tingaikin (Xored) - **New since last review**
+    
+    In addition, as the Network Service Mesh has grown, it has recently begun adding sub-MAINTAINERS to specific repos in the organization:
+    * Alexandre Menezes (Red Hat) - [nsm-operator](https://github.com/networkservicemesh/nsm-operator)
+    * Denis Tingajkin (Xored) - [fanout](https://github.com/networkservicemesh/fanout)
+    * Dimitrios Markou (Ericsson Software Technology) - [sdk-ovs](https://github.com/networkservicemesh/sdk-ovs), [cmd-forwarder-ovs](https://github.com/networkservicemesh/cmd-forwarder-ovs) - **New since last review**
+    * Periyasamy Palanisamy (Ericsson Software Technology) - [sdk-ovs](https://github.com/networkservicemesh/sdk-ovs), [cmd-forwarder-ovs](https://github.com/networkservicemesh/cmd-forwarder-ovs) - **New since last review**
+    * Laszlo Kiraly (Ericsson Software Technology) - [cmd-nse-remote-vlan](https://github.com/networkservicemesh/cmd-nse-remote-vlan) - **New since last review**
+    
+3.  _What do you know about adoption, and how has this changed since your last review / since you joined Sandbox? If you can list companies that are end users of your project, please do so. (Feel free to link to an existing ADOPTERS file if appropriate.)_
+
+    At least four companies are beginning to build this into next generation architecture:
+    * [Anthem](https://anthem.ai/)
```

Comment:
any chance you can link to the actual products these firms are building? I am keen to know personally, but it may not be public.

Reply:
@sdake I can only provide the authorized quotes below.